examples/platforms/posix/radio.c

```diff
@@ -531,7 +531,7 @@ void posixRadioUpdateFdSet(fd_set *aReadFdSet, fd_set *aWriteFdSet, int *aMaxFd)
 
 void posixRadioProcess(void)
 {
-    const int flags = POLLRDNORM | POLLERR | POLLNVAL | POLLHUP;
+    const int flags = POLLIN | POLLRDNORM | POLLERR | POLLNVAL | POLLHUP;
```

Comment:
I don't know about this stuff to say either way. Everything seems to work like this, so I'd vote to just use both flags.